packages/office-ui-fabric-react/src/components/List/List.Props.ts

```diff
@@ -75,14 +95,48 @@ export interface IListProps extends React.HTMLAttributes<List | HTMLDivElement>
 
   /** Number of items to render. Defaults to items.length. */
   renderCount?: number;
+
+  /**
+   * Called to add styles to a given cell during rendering.
+   */
+  getCellStyle?: (item: any, index: number) => any;
+
+  /**
+   * Called to add a class to a given page during rendering.
+   */
+  getPageClassName?: (page: IPage) => string;
+
+  /**
+   * Called to add a class to a given list cell during rendering.
+   */
+  getCellClassName?: (item: any, index: number) => string;
+
+  /**
+   * Additional class for the list surface element.
+   */
+  surfaceClassName?: string;
+
+  role?: string;
```

Comment:
Comments. Is this `role` mixed into the root? What is the default value? (presentation)